src/redux/selectors.js

```diff
@@ -17,13 +17,13 @@ const getCellSetsHierarchy = (...params) => (state) => (
 const getCellSetsHierarchyByType = (...params) => (state) => (
   cellSetsSelectors.getCellSetsHierarchyByType(...params)(state.cellSets));
 
-const getCellSetsHierarchyByKey = (...params) => (state) => (
-  cellSetsSelectors.getCellSetsHierarchyByKey(...params)(state.cellSets));
+const getCellSetsHierarchyByKeys = (...params) => (state) => (
+  cellSetsSelectors.getCellSetsHierarchyByKeys(...params)(state.cellSets));
 
 export {
   getBackendStatus,
   getCellSets,
   getCellSetsHierarchy,
   getCellSetsHierarchyByType,
```

Comment:
We should start grouping the selectors and export - `cellSetsSelectors` instead of defining every single one here. In the future this will become a big mess.

Reply:
This can already be done if we remove this file and import directly from `redux/selectors/{selectorGroup}`. I agree with you it's going to be messy in the long run. What do you think @cosa65?

Reply:
This can already be done if we remove this file and import it directly from `redux/selectors/{selectorGroup}`. I agree with you it's going to be messy in the long run. What do you think @cosa65

Reply:
I think this issue can be addressed in a different ticket.